peps/pep-0728.rst

```diff
@@ -554,42 +627,68 @@ Because it did not offer a way to specify the type of the extra items, the type
 checkers will need to assume that the type of the extra items is ``Any``, which
 compromises type safety. Furthermore, the current behavior of TypedDict already
 allows untyped extra items to be present in runtime, due to structural
-subtyping.
+subtyping. ``closed=True`` plays a similar role in the current proposal.
 
 Supporting ``TypedDict(extra=type)``
 ------------------------------------
 
-This adds more corner cases to determine whether a type should be treated as a
-type or a value. And it will require more work to support using special forms to
-type the extra items.
-
-While this saves us from reserving an attribute for special use, it will require
-extra work to implement inheritance, and it is less natural to integrate with
-generic TypedDicts.
+While this design is potentially workable, there are several partially
+addressable concerns to consider. Adding everything up, it is slightly less
+favorable than the current proposal.
+
+- Usability of forward reference
+  As in the functional syntax, using a quoted type or a type alias will be
+  required when SomeType is a forward reference. This is already a requirement
+  for the functional syntax, so implementations can potentially reuse that piece
+  of logic, but this is still extra work that the ``closed=True`` proposal doesn't
+  have.
+
+- Concerns about using type as a value
+  Whatever is not allowed as the value type in the functional syntax should not
+  be allowed as the argument for extra either. While type checkers might be able
+  to reuse this check, it still needs to be somehow special-cased.
+
+- Support for the functional syntax
+  Something like ``TD = TypedDict("TD", {"foo": str}, extra=SomeType)`` versus
+  ``TD = TypedDict("TD", {"foo": str, "_": SomeType}, closed=True)`` should be
+  allowed.
+
+- Handling Inheritance
```

Comment:
This one doesn't sound very convincing either. Especially now that `__extra_items__` can be both a regular and a special key, the inheritance behavior is not all that simple. I think this paragraph can just be removed.